Godeps/_workspace/src/k8s.io/kubernetes/pkg/apimachinery/registered/registered.go

```diff
@@ -24,8 +24,10 @@ import (
 	"strings"
```

Comment:
> Can I drop this in 1.4?

No.  You'll still need to have `oapi` function.  We're carrying this until those types not longer need registration.